.github/workflows/main.yml

```diff
@@ -37,6 +37,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+
+      - name: Set environment variable
+        run: |
+          export GISAID_API_TOKEN=${{ secrets.GISAID_API_TOKEN }}
```

Comment:
Github actions have a special env statement that can be directly added to the step which requires the variable, no need to use run for this.